src/Controllers/common.ts

```diff
@@ -147,28 +147,16 @@ export async function handleCreateByName(
 
   // Check if link object exsits
 
-  try {
-    // @ts-ignore
-    const linked = await prisma[link.type].findUnique({ where: { pid: link.id }, select: { id: true } });
-
-    if (!linked) {
-      return res.status(404).json({
-        type: "error",
-        payload: {
-          message: `Could not link to ${link.type} with ID '${link.id}'`,
-        },
-      });
-    }
-  } catch (e) {
-    // REVIEW: Check for valid UUID
-    if (e instanceof PrismaClientUnknownRequestError) {
-      return res.status(400).json({
-        type: "error",
-        payload: {
-          message: "Unknown error occured. This could be due to malformed IDs",
-        },
-      });
-    }
+  // @ts-ignore
+  const linked = await prisma[link.type].findUnique({ where: { pid: link.id }, select: { id: true } });
+
+  if (!linked) {
+    return res.status(404).json({
```

Comment:
Can you not just throw a 404 error here?

Reply:
Well, I think that a simple 404 response would make it harder to recognise, what went wrong. For mor consistancy, I will make LinkingFailedError (inspired by the NotFoundError). Good point tho

Reply:
Good Point. In this case it makes sense to throw a not found error. A linking error would make sense in a try-catch-block around the db-operation that actually connects them.